public/index.html

```diff
@@ -27,6 +27,7 @@
   <link rel="stylesheet" href="/app/engagementindex/leaderboard/userinfo.css" />
   <link rel="stylesheet" href="/app/engagementindex/points/points.css" />
   <link rel="stylesheet" href="/app/shared/activityTimeline.css" />
+  <link rel="stylesheet" href="/app/shared/apiError.css" />
```

Comment:
Thanks, @johncrossman, I was about due to revisit https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git ... amended.